index.html

```diff
@@ -7,172 +7,233 @@
 </head>
 
 <body>
-  <header class='page-header'>
-    <a class="logo" href="https://htmlacademy.ru/">
-      <img class='logo' src='#' alt='Логотип Седона'>
-    </a>
-    <nav class='navigation'>
-      <ul class='navigation-list'>
-        <li class='navigation-item'>
-          <a class='navigation-link' href='#'>Главная</a>
+  <header class="page-header">
+    <a class="logo">
+      <img class="logo" src="#" alt="Логотип Седона">
+     </a>
```

Comment:
Форматирование кажется чуть поехало
```suggestion
    <a class="logo">
      <img class="logo" src="#" alt="Логотип Седона">
    </a>
```